lib/rules/new-parens.js

```diff
@@ -31,31 +31,63 @@ module.exports = {
         },
 
         fixable: "code",
-        schema: [],
+        schema: {
+            anyOf: [
+                {
+                    type: "array",
+                    items: [
+                        {
+                            enum: ["always", "never"]
+                        }
+                    ],
+                    minItems: 0,
+                    maxItems: 1
+                }
+            ]
+        },
         messages: {
-            missing: "Missing '()' invoking a constructor."
+            missing: "Missing '()' invoking a constructor.",
+            using: "'()' included invoking a constructor with no arguments."
```

Comment:
Not sure about the messageId here. I also don't love starting a message with parentheses tokens, so I'd like to suggest an alternative wording:

```suggestion
            unnecessary: "Unnecessary '()' invoking a constructor with no arguments."
```